src/ui/studio/keyboard-shortcuts/globalKeys.js

```diff
@@ -0,0 +1,182 @@
+// Groups for displaying shortcuts in the overview page
+const recordGroup = 'record-shortcuts';
+const editGroup = 'edit-shortcuts';
+const otherGroup = 'other-shortcuts';
+
+const back = 'backspace-key'
+const enter = 'enter-key'
+const del = 'delete-key'
+const ctrl = 'control-key'
+const shift = 'shift-key'
+const space = 'space-key'
+const left = 'left-key'
+const right = 'right-key'
+
+export const keyMap = {
```

Comment:
Quite a bit of duplication in this array. Which is not a huge deal, but one could certainly make this smaller. An example:

```
const recordShortcuts = {
  RECORD_DISPLAY: {
    name: 'record-display',
    sequences: ['d'],
    description: ['d'],
  },
  // ...
};
const editShortcuts = /* ... */;

const mapValues = (obj, mapper) => Object.fromEntries(Object.entries(obj).map(mapper));

export const keyMap = {
  ...mapValues(recordShortcuts, ([key, value]) => 
    [key, { ...value, action: 'keydown', group: 'record-shortcuts' }]),
  ...mapValues(editShortcuts, ([key, value]) => 
    [key, { ...value, action: 'keydown', group: 'edit-shortcuts' }]),
  // ...
};
```

Whether thats really better ... :shrug: you decide